airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -111,15 +110,15 @@ def patch_dag(session, dag_id, update_mask=None):
 
 @security.requires_access([(permissions.ACTION_CAN_DELETE, permissions.RESOURCE_DAG)])
 @provide_session
-def delete_dag(dag_id: str, session: Session):
+def delete_dag(dag_id: str, session):
```

Comment:
Isn't this change the same as just `type: ignore` -- we should keep the `Session` or just ignore it for now so we can identifying the pending changes

Reply:
Yeah. Switching to the new method by @uranusjr :)